recipes/qdldl-python/patches/0001-Change-cmake-dir.patch

```diff
@@ -0,0 +1,26 @@
+From e5d44c3516fdea0940e9897b87132ca6331c4fcf Mon Sep 17 00:00:00 2001
+From: Bartolomeo Stellato <bartolomeo.stellato@gmail.com>
+Date: Wed, 3 Jun 2020 11:40:50 -0400
+Subject: [PATCH] Change cmake dir
+
+---
+ setup.py | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index e0e9288..35ec9c8 100644
+--- a/setup.py
++++ b/setup.py
+@@ -58,7 +58,8 @@ else:
+ 
+ # Compile QDLDL using CMake
+ current_dir = os.getcwd()
+-qdldl_dir = os.path.join(current_dir, 'c',)
++root_dir = os.path.dirname(os.path.abspath(__file__))
++qdldl_dir = os.path.join(root_dir, 'c')
```

Comment:
This might be why the C++ paths are messed up. Can you run conda-build locally?

Reply:
@scopatz thank you for the suggestion.

I tried both with and without that patch and it still gives the same problem. I thought that it was a path issue since before the error reported by @h-vetinari there is [this cmake error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=170289&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=600).

I can reproduce the errors locally on docker following [this command](https://conda-forge.org/docs/maintainer/adding_pkgs.html#staging-test-locally). Unfortunately, various combinations of patches haven't solved the issue for me. With standard `python setup.py install` it works perfectly.

Have you seen similar errors in other projects?